crates/ruff/resources/test/fixtures/pyupgrade/UP032_2.py

```diff
@@ -0,0 +1,5 @@
+"{.real}".format(1)
+
+"{0.real}".format(1)
+
+"{a.real}".format(a=1)
```

Comment:
Can you add some more test cases here, for floats, octal integers, complex, etc.?